### PR TITLE
Remove Br prefix from BearSSL

### PR DESCRIPTION
**Pull request by Menduist**
ref https://github.com/status-im/nim-bearssl/pull/27

### CODE REVIEW

File: libp2p/crypto/chacha20poly1305.nim
Comment (arnetheduck) [inline libp2p/crypto/chacha20poly1305.nim:20]:
```suggestion
import bearssl/blockx
```

File: libp2p/crypto/crypto.nim
Comment (arnetheduck) [inline libp2p/crypto/crypto.nim:72]: `intx` should be fixed in bearssl instead - ie it should be imported by any `abi/bearss_xxx` that requires access to `int` symbols - what was the error message exactly?

File: libp2p/protocols/pubsub/floodsub.nim
Comment (arnetheduck) [inline libp2p/protocols/pubsub/floodsub.nim:229]: this is unnecessary, there's an `openArray` version in `bearssl/rand`

**Comment by arnetheduck** [comment]
to really reap the benefits, all `import bearssl` should systematically be removed from the codebase and replaced with specific imports (and reexports, for public symbols)

**Comment by Menduist** [comment]
I think we can merge this once the bearssl PR is merged, and switch to the new `rng` once the generic system is done

File: libp2p/crypto/ed25519/ed25519.nim
Comment (arnetheduck) [inline libp2p/crypto/ed25519/ed25519.nim:16]: should export `rand` here too since there's a public symbol that references it - this way, the tests and examples no longer explicitly have to import `bearssl` and it can be removed completely, making the lib easier to use

**Review by arnetheduck — DISMISSED** [review]

**Comment by arnetheduck** [comment]
one more `bearssl` in `libp2p.nim`

**Review by arnetheduck — APPROVED** [review]

**Pull request merged**
